src/Microsoft.Extensions.Configuration.AzureAppConfiguration/FeatureManagement/FeatureManagementKeyValueAdapter.cs

```diff
@@ -26,13 +27,107 @@ public FeatureManagementKeyValueAdapter(FeatureFilterTracing featureFilterTracin
 
         public Task<IEnumerable<KeyValuePair<string, string>>> ProcessKeyValue(ConfigurationSetting setting, Uri endpoint, Logger logger, CancellationToken cancellationToken)
         {
+            _isMicrosoftSchema = false;
```

Comment:
I'm expecting this to be on a per flag basis.

Reply:
Otherwise adding an unrelated flag that has variants will cause old flags that are working to break.

Reply:
ProcessKeyValue is invoked once per flag. So each flag will either use .NET schema or MS schema. Am I missing something?

Reply:
You're right. The code would function correctly. I was thrown off because instance state is being used to flow a condition that's only relevant to the method invocation. To avoid that, the decision of which schema to use should be done by the one who calls parse feature flag and does not need to be saved as instance state.